transaction-service/src/main/java/pl/adamboguszewski/transaction/service/application/Transaction.java

```diff
@@ -46,4 +47,12 @@ public static Transaction fromRequest(CreateTransactionRequest request) {
                 products
         );
     }
+
+    public static Transaction fromDto(TransactionDto transactionDto) {
+        return new Transaction(
+                transactionDto.getTransactionId(),
+                transactionDto.getTotalPrice(),
+                transactionDto.getTransactionInformation(),
```

Comment:
Map transactionDto like we did map request in fromRequest, something like this: TransactionInformation.fromDto(transactionDto.getTransactionInformation())
and same thing with products